docs/gaia/join-testnet.md

```diff
@@ -103,6 +103,37 @@ curl https://raw.githubusercontent.com/cosmos/testnets/master/latest/genesis.jso
 Note we use the `latest` directory in the [testnets repo](https://github.com/cosmos/testnets)
 which contains details for the latest testnet. If you are connecting to a different testnet, ensure you get the right files.
 
+You now need to generate a `genesis tx` to generate a signed self-delegation:
```

Comment:
Unless I'm mistaken, this entire section up to "Add seed nodes" belongs in `validator-setup.md` and not `join-testnet`. That's because this workflow only concerns people that intend to run validator nodes, not non-validator full-nodes.

Reply:
I've made some amendments to `validator-setup.md`, please have a look at it :+1:

Reply:
My comment still stands, the part from `You now need to generate a genesis tx to generate a signed self-delegation:` to `The previous command will collect all genesis transactions and modify genesis.json accordingly.` does not belong in this file. The `join-testnet` section is only about people that want to run a regular full node. They only need to know how to fetch the genesis file and sync from there. They don't need to know how to generate a `genesis-tx`.